compiler/src/iree/compiler/Codegen/Common/GPU/GPUPromoteMatmulOperands.cpp

```diff
@@ -95,7 +118,7 @@ void promoteResult(OpBuilder &builder, Operation *op, Value valToMakeShared) {
   }
 
   rewriter.setInsertionPointAfterValue(replacement);
-  replacement = promoteValue(rewriter, loc, replacement);
+  replacement = promoteValue(rewriter, loc, replacement, false);
```

Comment:
style nit (it also helps review): add argument comment when it is a magic value.

```suggestion
  replacement = promoteValue(rewriter, loc, replacement, /*useDirectLoad=*/false);
```